mlir-tensorrt/compiler/include/mlir-tensorrt/Compiler/StableHloToExecutable.h

```diff
@@ -128,6 +128,9 @@ struct StableHLOToExecutableOptions : public mlir::OptionsContext {
   /// Whether to disallow host tensors in TensorRT clusters.
   bool disallowHostTensorsInTensorRTClusters = false;
 
+  /// Whether to use non-DPS style calling convention.
```

Comment:
This should say something along the lines of "use non-DPS calling convention for the entrypoint function and backend types that support allocating results"